packages/forms/src/Components/RichEditor/RichContentRenderer.php

```diff
@@ -235,19 +241,29 @@ public function getTipTapPhpExtensions(): array
             app(CodeBlock::class),
             app(CustomBlockExtension::class),
             app(Document::class),
+            app(HardBreak::class),
             app(Heading::class),
+            app(Highlight::class),
+            app(HorizontalRule::class),
             app(Italic::class),
             app(ImageExtension::class),
+            app(LeadExtension::class),
             app(Link::class),
             app(ListItem::class),
             app(MergeTagExtension::class),
             app(OrderedList::class),
             app(Paragraph::class),
             app(RenderedCustomBlockExtension::class),
+            app(SmallExtension::class),
             app(Strike::class),
             app(Subscript::class),
             app(Superscript::class),
             app(Text::class),
+            new TextAlign([
+                'types' => ['heading', 'paragraph'],
+                'alignments' => ['start', 'center', 'end', 'justify'],
+                'defaultAlignment' => 'start',
+            ]),
```

Comment:
Please use the `app()` helper so that users can swap this class out for their own in the container

Reply:
Moved this out to a dedicated class so it can be used with `app()`. If you prefer to used the default extension we can, but the default uses 'left' and 'right' which don't work with rtl and would mean more complicated integration. By using 'start' and 'end' support for rtl works without any complicated logic.

Reply:
You don't need a dedicated class for it to be used with `app()`, you can pass an array of params in the second arg of `app()`